rosdep/base.yaml

```
@@ -2262,6 +2262,11 @@ libann-dev:
   debian: [libann-dev]
   fedora: [ann-devel]
   ubuntu: [libann-dev]
+libao-dev:
+  alpine: [libao-dev]
+  debian: [libao-dev]
+  fedora: [libao-devel]
```

```suggestion
  fedora: [libao-devel]
  opensuse: [libao-devel]
  rhel: [libao-devel]
```

Add suggested targest from linter

---

That's a great suggestions. Thanks!